README.md

```diff
@@ -4,10 +4,21 @@ This is a basic chat bot framework for Beam, built to serve as a basic example o
 
 ## Usage
 
-We've built a chat bot that logs incoming messages to the console, and sends "Hi!" to the channel every second.
+We've built a chat bot that logs incoming messages to the console, no longer sends "Hi!" to the channel every second.
+!ping command has been added
 
 To use it, ensure Python 3 is installed on your system. Then:
 
-- install dependencies (`python setup.py install`)
-- copy `config.example.py` to `config.py` and add your authentication information
-- run the example chatbot (`python example.py`)
+* install dependencies
+  * Windows : `python setup.py install`
+  * Linux   : `sudo python setup.py install`
```

Comment:
This is not necessary if you use a virtualenv (generally best practice) or use a non-privileged install (`/usr/local/...`). I'd get rid of this.